presentation/src/main/java/com/anytypeio/anytype/presentation/home/HomeScreenViewModel.kt

```diff
@@ -163,7 +165,8 @@ class HomeScreenViewModel(
     private val getSpaceView: GetSpaceView,
     private val searchObjects: SearchObjects,
     private val getPinnedObjectTypes: GetPinnedObjectTypes,
-    private val userPermissionProvider: UserPermissionProvider
+    private val userPermissionProvider: UserPermissionProvider,
+    private val notificationsChannel: NotificationsChannel
```

Comment:
no longer needed?

Reply:
yes, all Notifications logic moved to Main View Model for now